public/angular/sass/app.scss

```diff
@@ -43,6 +56,14 @@ a {
   margin-right: 10px;
 }
 
+.margin-left {
+  margin-left: 10px;
+}
+
+.margin-bottom-lg {
+  margin-bottom: 20px;
+}
+
```

Comment:
A lot of the class names here are un-semantic (they don't describe the content of the element, just the style). I always was taught this was not such a good practice, but maybe they're doing things differently nowadays?